src/algorithms/primeFinder.js

```diff
@@ -0,0 +1,19 @@
+export default function isPrime(number) {
+    var start = Math.ceil(number / 2);
+    for (let i = start; i >= 2; i--) {
+        if (!(number%i)) {
+            return false;
+        }
+    }
+    return true;
+}
+export function primeFactors(number) {
+    var start = Math.ceil(number / 2);
+    var factors = {};
+    for (let factor = start; factor > 1; factor--) {
+        if (!(number%factor) && isPrime(factor)) {
```

Comment:
I'd recommend spaces here. I think when i told you about the syntax I didn't include spaces, so that's my bad, but i think spaces make it more readable.
```suggestion
        if (!(number % factor) && isPrime(factor)) {
```